ecs/daemon/inputs/task.conf

```diff
@@ -0,0 +1,13 @@
+[INPUT]
+    Name                tail
+    Tag                 app.*
+    Path                /var/lib/docker/containers/*/*.log
```

Comment:
I've tested that if I add:

```
    Exclude_Path        *container-cached*
```

This will ignore the container cache log files, so we only pick up containers that explicitly configure json-file log driver.

Reply:
Except, I saw on my instance that the ECS Agent seems to somehow have non-cache log files, I am not sure how this happened:

<img width="1792" alt="Screen Shot 2023-01-21 at 9 28 18 PM" src="https://user-images.githubusercontent.com/29443996/213902192-dccb5598-8ad6-41c8-8473-61e9d4746a2d.png">

Reply:
This shows that it does indeed work for real cache files: see the excluded debug message: 
<img width="1792" alt="Screen Shot 2023-01-21 at 9 33 27 PM" src="https://user-images.githubusercontent.com/29443996/213902196-331a36d8-a588-4a35-ba7d-ac151957def7.png">

Reply:
I did a simplified test where I scanned one container sub-directory on my test instance ^

Reply:
I checked the agent thing on another instance, it seems that the agent typically has json file log driver files. This is strange and unexpected. 

I should look into this/ask ECS Agent team if this is on purpose. I don't think it is.

Reply:
https://github.com/aws/amazon-ecs-agent/issues/3546